doc/news/_preparation_next_release.md

```diff
@@ -367,6 +367,14 @@ removed due to:
 - New plugin to validate hex formatted colors (e.g. #fff or #abcd) and normalize them to rgba (4294967295 (= 0xffffffff) and 2864434397 (= 0xaabbccdd) respectively). It also has support for named colors according to the [extended color keywords](https://www.w3.org/TR/css-color-3/#svg-color) from CSS3.
   _(Philipp Gackstatter)_
 
+### Ini
+
+- new formatting is introduced --> instead of having whitespaces around '=' now the whitespaces are not allowed anymore
```

Comment:
The whitespaces should be allowed (it would be an incompatible change to disallow it) but the INI plugin should not write out these whitespaces anymore.

Reply:
@markus2330 Thank you for your comment.
Well, the problem is that I started working on this issue since months and nobody has told me the whitespaces should be allowed. I had another pull request where I pushed my changes I think at the beginning of April. These changes clearly showed that the whitespaces will not be allowed anymore. It would've saved me a lot of time since I hadn't had to fix all those tests due to different formatting. Now I have to reimplement it, probably again fix the tests, etc. I am very much disappointed...

Reply:
I am sorry for the confusion but #2065 clearly says that something additional should be supported and it does not mention that previous working files should be rejected.

I do not think you need to rewrite all tests, nor that your changes are useless. In all tests where the INI file is **written** you obviously need to remove the spaces, otherwise the files would not be identical after writing.

Reply:
@markus2330 
So, after spending many many hours, this is my outcome:
1. Reading is possible both with and without spaces. The example below will not cause any problems:
```
one=1
two = 2
```
2. Writing is only done without spaces -- if a file had mixed approaches like 
```
one=1
two = 2
```
after writing for example --

> > set user/test/three 3 

the result would be
```
one=1
two=2
three=3
```
Do you confirm this kind of behaviour is what it should be? 

@markus2330 Can you please reward my group colleague @e01306821 with some extra points for the class? He provided me support multiple times and deserves nothing less than a "1" in my opinion. Sorry for the off-topic. =)

Reply:
> Do you confirm this kind of behaviour is what it should be?

Yes, perfect. This should fix #2541.

> Can you please reward my group colleague @e01306821 with some extra points for the class?

Yes. Please try to reflect what you learned in the tutorials/docu.

Reply:
Yes, I will! Thank you very much!